src/basic/lines.js

```diff
@@ -14,19 +14,30 @@
 	EXTERNAL DEPENDENCIES
  */
 
-import {curry, compose, not, equals, identity, cond} from 'ramda';
+const ramda = require('ramda');
```

Comment:
Hi, just trying to help! I don’t know the reason about going back to require but, there is a sexier way of having this, better than several const, using decomposition!

Also, importing the full ramda you’ll add 250kb of unminified size to the library

```javascript

const { not, curry, compose, ... } = ramda
```